assets/pages/monthly-contributions/monthlyContributions.jsx

```diff
@@ -41,7 +41,12 @@ const store = createStore(reducer, {
 
 
 // Retrieves the contrib amount from the url and sends it to the redux store.
-store.dispatch(setContribAmount(getQueryParameter('contributionValue', '5')));
+const contributionAmount = getQueryParameter('contributionValue', '5');
```

Comment:
This is a value returned by our function right? Why do we have to check for undefined and null? Can we make it always return null in the situation where there is no query param and no default value set?